apps/dapp-console-api/vitest.config.ts

```diff
@@ -12,5 +12,6 @@ export default defineConfig({
     testTimeout: 20_000,
     globalSetup: ['./src/testUtils/global.ts'],
     setupFiles: ['./src/testUtils/setup.ts'],
+    reporters: 'verbose',
```

Comment:
did you mean to leave this in or were you just using it for debugging?

Reply:
oh let me remove was for debugging